x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/mapper/SemanticTextInferenceResultFieldMapper.java

```diff
@@ -261,6 +263,7 @@ private static NestedObjectMapper createNestedObjectMapper(
             SPARSE_VECTOR_SUBFIELD_NAME,
             ObjectMapper.Defaults.SUBOBJECTS
         ).add(
+            // TODO: How to add this field mapper without indexing the value?
             new BooleanFieldMapper.Builder(SparseEmbeddingResults.Embedding.IS_TRUNCATED, ScriptCompiler.NONE, false, indexVersionCreated)
```

Comment:
If you don't want to index it & you want it to exist besides it being in `_source` (which it already is), you can use a stored field.

Reply:
I don't think we need that as stored field - just that it is included as part of _source. Would there be a point on it being a stored field?